clients/privacy-center/cypress/e2e/consent-events.cy.ts

```diff
@@ -0,0 +1,640 @@
+import type { FidesEventDetail, FidesEventType } from "fides-js/src/lib/events";
+
+import { stubConfig, stubTCFExperience } from "../support/stubs";
+
+interface FidesEventExpectation {
+  type: FidesEventType;
+  detail: Partial<FidesEventDetail>;
+}
+
+describe("Consent FidesEvents", () => {
```

Comment:
New spec file that exercises both banner+modal and TCF components, triggering all the FidesEvents along the way.

It's hard to reason about this behaviour from the other specs that are much more focused on the core logic of those components instead of the events they trigger. There's some duplicate tests now, but more coverage isn't necessarily bad.